tests/plugins/test_dependencies.py

```diff
@@ -192,3 +192,47 @@ def test_dependency_missing_newline(self):
                 "not be found within the VTs.",
                 results[0].message,
             )
+
+    def test_inline_comment_dependency(self):
+        with self.create_directory() as tmpdir:
+            path = tmpdir / "file.nasl"
+            example = tmpdir / "common" / "example.inc"
```

Comment:
Better use the `.nasl` file ending here and in the other test as `script_dependencies()` doesn't use `.inc` files.

Reply:
Done